packages/viem/package.json

```
@@ -0,0 +1,54 @@
+{
+  "name": "@eth-optimism/fee-estimation",
```

Would this be `@eth-optimism/viem` or something different?

---

```suggestion
  "name": "@eth-optimism/viem",
```
I never have opinionhs on naming things but viem would be my first guess. Could just call it `@eth-optimism/sdk` too as it is the next generation of the sdk and more than other implementations our official one